apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -165,7 +165,7 @@ public static IdDt buildPatientId(Beneficiary beneficiary) {
    *     Beneficiary}
    */
   public static IdDt buildPatientId(Long beneficiaryId) {
-    return new IdDt(Patient.class.getSimpleName(), beneficiaryId.toString());
+    return new IdDt(Patient.class.getSimpleName(), beneficiaryId);
```

Comment:
I remember this change from a previous PR. Why the change back?

Reply:
This is the proverbial "more than one way to skin a cat". Now for something like this I prefer to let the class itself handle things; to wit, here are the 2 pertinent constructors:

IdDt(String theResourceType, Long theIdPart)
IdDt(String theResourceType, String theId)

So in this case, I think it's better to pass the Long and let the class handle it rather than do some gyration in the caller to 'fit it in'.

Reply:
This is probably academic at this point since I agree with this change. However this is what it looked like originally and you modified it at the very end of 1532 because you found something in your testing. I just want to be certain that there is not something here of concern as I didn't understand originally why you wanted to change it.

Reply:
no, did not find anything other than doing some QC of code and I knew from previous work (BFD1076) that the FHIR IdDt class supported Long.